config.json

```diff
@@ -1432,8 +1443,8 @@
       "unlocked_by": null,
       "difficulty": 2,
       "topics": [
-          "conditionals",
-          "math"
+        "conditionals",
+        "math"
       ]
     },
     {
```

Comment:
I'm less concerned with making the additional edits to pass `configlet lint` here than I am with the fact that those violations made it through numerous CI builds without `configlet lint` properly flagging them. It's possible we are not invoking `configlet lint` correctly in Travis.

Reply:
Shall I keep those changes or I just change them back?

Reply:
Keeping them is fine. I was mainly making a point of reference for another issue to point to.